draft-ietf-jsonpath-base.md

```diff
@@ -382,41 +384,41 @@ of bracket notation. Examples and descriptions use shorthands where convenient.
 
 ### Selectors
 
-A wildcard `*` ({{wildcard}}) in the expression `[*]` selects all children of an
-object or an array and in the expression `..[*]` selects all descendants of an object or an array.
+A wildcard `*` ({{wildcard}}) in the expression `[*]` selects all children of a
+node and in the expression `..[*]` selects all descendants of a node.
 
 An array slice `start:end:step` ({{slice}}) selects a series of
 elements from an array, giving a start position, an end position, and
-possibly a step value that moves the position from the start to the
+an optional step value that moves the position from the start to the
 end.
 
-Filter expressions `?<boolean expr>` select certain children of an object or array as in
+Filter expressions `?<boolean expr>` select certain children of a value, as in:
```

Comment:
The old text said that filter expressions are useful for containers; I think that information should stay there (and, if not, we should say node, not value).

Reply:
Good point. Fixed.